spec/javascripts/Users/components/PermissionsForm.spec.jsx

```diff
@@ -0,0 +1,235 @@
+// @flow
```

Comment:
It's no needed for specs. Up to you ;)

Reply:
we might want to benchmark the time of execution for the tests with and w/o it.

Reply:
I'd say we should remove it, and also remove it from webpacker for our test folder. The latter can be done in a different PR, since it doesn't add any value to our tests, only time. Our flow already has a specific task.